stock-tracker/src/socket/subscriptions.ts

```diff
@@ -0,0 +1,23 @@
+import { socketService } from ".";
+import { eventActions } from "./eventActions";
+import { Dispatch } from "redux";
+
+type Payload = {
+  isError: boolean;
+  data: any;
```

Comment:
Q: why the data type is any?

Reply:
This would be all the data my application could receive. Since I have not yet typed all my components and data structures I have left this as 'any'.
However, soon when I finish I will create a global Payload type who's data would be a union of all the different data structures I have in this application.

Reply:
hint: turn Cindy's comment into an `issue`, so you don't forget to change this later on